internal/bmc/nsq.go

```diff
@@ -70,7 +70,11 @@ func (b *BMCService) HandleMessage(message *nsq.Message) error {
 		return err
 	}
 
-	b.log.Debug("got message", "topic", b.machineTopic, "channel", mqChannel, "event", event)
+	if message.Attempts > 3 {
```

Comment:
Through the `MaxAttempts` option, the client will already discard a message. It defaults to 5, so I am not certain if this is required.